test/specs/rules/require-dictionary-words.js

```diff
@@ -0,0 +1,204 @@
+var Checker = require('../../../lib/checker');
```

Comment:
You commented that there was a nice error message for people trying to use a word list that wasn't installed. There should probably be a test that verifies that.